addons/website_slides_survey/models/survey_survey.py

```diff
@@ -1,11 +1,42 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import models, fields, api
 
 
 class Survey(models.Model):
     _inherit = 'survey.survey'
+    slide_ids = fields.One2many('slide.slide', 'survey_id', 'Certifications')
```

Comment:
I would change the string a bit to `string="Certification Slides"`.
Because a "certification" is actually an instance of survey.survey.

I would also add a `help="The slides this survey is linked to through the e-learning application."`
It will help the end user if we use this field somewhere else and he hovers on it, and it will also help the next developer that reads this field and tries to figure out what it does.

(To avoid having lines of code that are too long, you can put the help attribute after a carriage return).

Reply:
Done with with 2 carriage returns to avoid PEP8 "E128 continuation line under-indented for visual indent" error